lua/lazyvim/config/keymaps.lua

```diff
@@ -10,6 +10,10 @@ map({ "n", "x" }, "<Down>", "v:count == 0 ? 'gj' : 'j'", { desc = "Down", expr =
 map({ "n", "x" }, "k", "v:count == 0 ? 'gk' : 'k'", { desc = "Up", expr = true, silent = true })
 map({ "n", "x" }, "<Up>", "v:count == 0 ? 'gk' : 'k'", { desc = "Up", expr = true, silent = true })
 
+-- Open folds on left/right
+map("n", "h", "foldclosed('.') > 0 ? 'zoh' : 'h'", { expr = true, silent = true })
+map("n", "l", "foldclosed('.') > 0 ? 'zol' : 'l'", { expr = true, silent = true })
```

Comment:
```suggestion
map("n", "h", "foldclosed('.') > 0 ? 'zo' : 'h'", { expr = true, silent = true })
map("n", "l", "foldclosed('.') > 0 ? 'zo' : 'l'", { expr = true, silent = true })
```
I don't believe there's a need for the direction key after `zo`. Only that seems to be working fine with `foldclosed()`.